TOPAS/Plots/2GySOBP/z-plot.py

```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from topas2numpy import BinnedResult
+
+files = []
+files.append(BinnedResult('/home/villads/2022_DCPT_LET/TOPAS/output/protons.592479/Scoring_protonD_ZBox.csv'))
```

Comment:
please avoid storing paths to local folders, the code should work without any modifications by any user